pokemongo_bot/cell_workers/move_to_fort.py

```diff
@@ -8,6 +8,8 @@
 
 class MoveToFort(BaseTask):
     def should_run(self):
+        if not self.bot.has_space_for_loot():
+            logger.log("Not moving to any forts as there aren't enough space. You might want to change your config to recycle more items if this message appears consistently.", 'yellow')
```

Comment:
I think you can just return False after this line in both workers and you'll have the behavior we are looking for.